index.html

```diff
@@ -51,212 +56,261 @@ <h1 class="hero__heading poppins">
+                </div>
             </div>
+
         </div>
 
         <div class="project">
 
-            <img src="images/multipost_stories.svg" alt="project">
-
-
-            <h1 class="project__heading poppins">
-                Multi-Post<br> Stories
-            </h1>
+            <img src="images/multipost_stories.svg" class="mobile" alt="project">
+            <img src="images/professional_printing_full.svg" class="full reverse" alt="project">
 
-            <ul class="project__path poppins">
-                <li class="project__path__name poppins">CANOPY</li>
-                <li class="project__path__type poppins">BACK END DEV</li>
-                <li class="project__path__type poppins">2015</li>
-            </ul>
+            <div>
+                <h1 class="project__heading poppins">
+                    Multi-Post Stories
+                </h1>
+
+                <ul class="project__path mobile poppins">
+                    <li class="project__path__name poppins">CANOPY</li>
+                    <li class="project__path__type poppins">BACK END DEV</li>
+                    <li class="project__path__type poppins">2015</li>
+                </ul>
 
-            <p class="project__desc poppins">
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
-            </p>
+                <ul class="project__path poppins full">
+                    <li class="project__path__name poppins">FACEBOOK</li>
+                    <li class="project__path__type poppins">Full Stack Dev</li>
+                    <li class="project__path__type poppins">2015</li>
+                </ul>
 
-            <ul class="project__language">
-                <li class="poppins">html</li>
-                <li class="poppins">css</li>
-                <li class="poppins">javaScript</li>
-            </ul>
+                <p class="project__desc poppins">
+                    <span class="mobile">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</span>
+                    <span class="full">Experimental content creation feature that allows users to add to an existing
+                        story over the course of a day without spamming their friends.</span>
+                </p>
+
+                <ul class="project__language">
+                    <li class="poppins">html</li>
+                    <li class="poppins full">Ruby on rails</li>
+                    <li class="poppins">css</li>
+                    <li class="poppins">javaScript</li>
+                </ul>
 
-            <div>
-                <button type='button' class="project__button poppins">
-                    see Project
-                </button>
+                <div>
+                    <button type='button' class="project__button poppins">
+                        see Project
+                    </button>
+                </div>
             </div>
 
         </div>
```

Comment:
- [ ] Some project cards are not responsive and their images do not resize. This is causing the content of the project cards to be displayed outside of the card's boundaries and out of the screen on screen sizes close to the 768px transition. This is a bad visual design. Please review the sizing of elements inside the card so they can be responsive. 
 -Your design:
<img width="823" alt="image" src="https://user-images.githubusercontent.com/61919617/186718232-6d776257-c962-4e5f-9a1f-55df8c8c2261.png">